onnx/test/helper_test.py

```diff
@@ -235,6 +237,44 @@ def test_model_docstring(self):
         self.assertFalse(hasattr(model_def, "name"))
         self.assertEqual(model_def.doc_string, 'test')
 
+    def test_model_version_default(self):
+        graph = helper.make_graph([], "my graph", [], [])
+        model_def = helper.make_model(graph)
+        v = helper.read_version(model_def.ir_version)
+        # it's fine to change this to be more lenient in the future
+        # right now we're interested in testing correct values are masked by default
+        self.assertEqual(v[0], 0)
+        self.assertEqual(v[1], 0)
+        self.assertEqual(v[2], 4)
+
+    def test_model_version_help(self):
+        v = helper.read_version(0x00000004)
+        self.assertEqual(v, (0, 0, 4))
+        v = helper.read_version(0x01100004)
+        self.assertEqual(v, (1, 16, 4))
+
+    def test_model_version_serialized(self):
+        graph = helper.make_graph([], "my graph", [], [])
+        model_def = helper.make_model(graph)
+        # all of these (valid) versions should yield the same serialization prefix
+        # note that version doesn't affect the prefix, so a couple of checks
+        # for stability will do
+        # 0D <- fixed32 field 1
+        # 4F <- [ 0- 7] bits of fixed value
+        # 4E <- [ 8-15] bits of fixed value
+        # 58 <- [16-23] bits of fixed value
+        # 46 <- [24-32] bits of fixed value
+        test_vers = [0x00000004, 0x01010004]
+        for t in test_vers:
+            model_def.ir_version = t
+            graph.name = "some name " + str(t)
+            s = model_def.SerializeToString()
```

Comment:
NB: This is testing an implementation detail of the protobuf library we're using; the library is allowed not to do this.